test/testdata/goldens/help_test/pub bump --help.txt

```diff
@@ -0,0 +1,17 @@
+# GENERATED BY: test/help_test.dart
+
+## Section 0
+$ pub bump --help
+Increases the version number of the current package.
+
+
+Usage: pub bump <subcommand> [arguments...]
+-h, --help        Print this usage information.
+    --major       Increment the major version number (eg. 3.1.2 -> 4.0.0)
+    --minor       Increment the minor version number (eg. 3.1.2 -> 3.2.0)
+    --patch       Increment the patch version number (eg. 3.1.2 -> 3.1.3)
+    --breaking    Increment to the next breaking version (eg. 0.1.2 -> 0.2.0)
```

Comment:
I think these should be subcommands instead, since:
 * you can't combine them (can you?); AND;
 * you offer do default value.

They don't feel like options, but more like separate actions one can take.

I think we could consider giving each of these subcommands a `--message` that would then flow into a `CHANGELOG` entry. If we wanted to.

I could imagine doing:
```console
$ dart pub bump major -m 'Initial release'
```

But then again, an option isn't really a super suitable place to write a CHANGELOG message.

Could be we just add a new section to the CHANGELOG by prepending it `"# v{{version}}\nTODO[block-publish]: Write changelog entry.\n\n"`

Then we could block publishing if `CHANGELOG.md` contains `TODO[block-publish]` (or maybe just `TODO`, perhaps `[block-publish]` is unnecessary)

Reply:
I think a message can come as a later feature. It requires parsing changelog, deciding on the format, updating the structure etc.

Reply:
Agreed!